.gitignore

```diff
@@ -2,4 +2,4 @@ typings/
 node_modules/
 test/bundle.d.ts
 *.js
-
+.idea/
```

Comment:
I don't want any editor specific ignores in the repo. You can add this to your own `.git/info/exclude`.